app/src/util/networks.ts

```diff
@@ -67,6 +70,8 @@ interface KnownTokenData {
   order: number
 }
 
+export const pseudoNativeAssetAddress = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
Out of curiousity, is there any particular reason you capitalized the address like this?

Reply:
I copied this from the 0x Project I think 🤔 and this is the checksum of that address, for us we could use any value here though cause we don't use the address in any transactions.